src/soilbiogeochem/SoilBiogeochemNitrogenFluxType.F90

```diff
@@ -334,27 +334,25 @@ subroutine InitHistory(this, bounds)
 
     do l = 1, ndecomp_cascade_transitions
        ! vertically integrated fluxes
-       !-- mineralization/immobilization fluxes (none from CWD)
-       if ( .not. decomp_cascade_con%is_cwd(decomp_cascade_con%cascade_donor_pool(l)) ) then
```

Comment:
I'm a little stuck wondering what we should do here... I wasn't imagining this flux would be associated with N mineralization or immobilization at this point.  Maybe it should remain with the comment that there is none from CWD